src/spaces/cell_complex.rs

```diff
@@ -0,0 +1,201 @@
+#![allow(dead_code)]
+
+use std::{collections::HashSet, hash::Hash, rc::Rc};
+
+use crate::topology::{OpenSet, PreSheaf, Section, Sheaf, TopologicalSpace};
+
+#[derive(Clone, Hash, PartialEq, Eq)]
+pub enum Wrapper<T: Eq + Hash + Clone> {
+    Point(T),
+}
+
+// Trait for an n-cell in a cell complex. Inherits from TopologicalSpace, and adds the cell's identification map.
+pub trait Cell<T: Eq + Hash + Clone>:
+    TopologicalSpace<Point = Wrapper<T>, OpenSet = HashSet<Wrapper<T>>>
+{
+    fn cell_points(&self) -> Vec<<Self as TopologicalSpace>::Point> {
+        self.points().into_iter().collect()
+    }
+    fn cell_point_neighborhood(
+        &self,
+        point: <Self as TopologicalSpace>::Point,
+    ) -> Vec<<Self as TopologicalSpace>::Point> {
+        self.neighborhood(point).into_iter().collect()
+    }
+    fn identification(&self, skeleton: &Skeleton<T>) -> HashSet<<Self as TopologicalSpace>::Point>; //Should return a set of points in the cell identified with previous skeleton points, and thus shouldn't be included in the next skeleton.
+}
+
+// A skeleton is a collection of cells, glued together by their identification maps.
```

Comment:
Feel free to make these doc comments with `///`

[See more here](https://doc.rust-lang.org/reference/comments.html#:~:text=Comments%20follow%20the%20general%20C%2B%2B,as%20a%20form%20of%20whitespace.)